### DISCUSSION TITLE
Propose to add CyberNvim to Traap/nvims

### DISCUSSION THREAD

**Discussion by thyttan** [pgosar/useful-things#6] [Ideas]
...as per: https://github.com/Traap/nvims#so-you-want-to-add-your-neovim-configuration-so-others-can-use-it

**Comment by pgosar** [comment]
I'm definitely fine with CyberNvim being added to this, feel free to.

**Reply by thyttan** [reply]
PR pending on https://github.com/Traap/nvims/pull/7
